### PR TITLE
Add from master lists improvements

### LINKED ISSUE #415: Add from master list: tidy-ups
Some issues spotted
- [x] Inbound: After adding, the lines show in blue.. but not if the 'group by item' option is selected
- ~Outbound: If you add from master list, then click 'Allocate placeholder lines' all the blue lines disappear. That's odd, right?~
- ~Inbound: lines with zero quantity are removed when saved, and maybe they shouldn't be ( see [this comment](https://github.com/openmsupply/open-msupply/pull/374#issuecomment-1198912196) )~
- [x] Outbound: You can add from master list when the status isn't new ( see [this comment](https://github.com/openmsupply/open-msupply/pull/374#pullrequestreview-1055063596) )

### PR DESCRIPTION
**Pull request by mark-prins**
Closes #415

Just a few misc items. I've decided not to address the line removal issues - the behaviour is a little odd in the context of placeholder rows, but is consistent for normal rows. Thought to leave the server changes for later, so that we can discuss properly before implementing those.

Allowing editing of the placeholder rows on inbounds will require changes to the server - this currently throws an error if you try to edit, which is why the f/e is deleting those lines and not allowing edits.

The removal when allocating lines in outbounds is also a deliberate implementation, so have left that for now too.

### CODE REVIEW

File: client/packages/invoices/src/InboundShipment/DetailView/ContentArea/columns.ts [hunk 1]
```diff
@@ -229,15 +220,15 @@ export const useInboundShipmentColumns = () => {
               const { lines } = rowData;
               return ArrayUtils.ifTheSameElseDefault(lines, 'packSize', '');
             } else {
-              return getPackSize(rowData) ?? '';
+              return rowData.packSize ?? '';
```
Comment (mark-prins) [inline client/packages/invoices/src/InboundShipment/DetailView/ContentArea/columns.ts:223]: added some of these back in for placeholder lines, to make the view consistent between grouped and not

File: client/packages/invoices/src/OutboundShipment/DetailView/AddFromMasterListButton.tsx [hunk 2]
```diff
@@ -4,13 +4,15 @@ import {
...
   PlusCircleIcon,
+  InvoiceNodeStatus,
 } from '@openmsupply-client/common';
 import { MasterListSearchModal } from '@openmsupply-client/system';
 import { useOutbound } from '../api';
 
 export const AddFromMasterListButtonComponent = () => {
   const t = useTranslation('distribution');
-  const isDisabled = useOutbound.utils.isDisabled();
+  const { status } = useOutbound.document.fields(['status']);
```
Comment (mark-prins) [inline client/packages/invoices/src/OutboundShipment/DetailView/AddFromMasterListButton.tsx:14]: disable if the status is not 'new'

File: server/configuration/example.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@
 #       http://localhost:3003,
 #       https://demo-open.msupply.org,
 #       http://localhost:8000,
-#     ] # Used to set the allowed Origin in Cross Origin Requst Security
+#     ] # Used to set the allowed Origin in Cross Origin Request Security
```
Comment (mark-prins) [inline server/configuration/example.yaml:13]: offroad nit pick

File: server/service/src/report/default_queries.rs [hunk 4]
```diff
@@ -63,6 +63,9 @@ query InvoiceQuery($storeId: String, $dataId: String) {
           itemCode
           itemId
           itemName
+          item {
```
Comment (mark-prins) [inline server/service/src/report/default_queries.rs:66]: was using this in the pick list report, so have updated the default query

File: server/service/src/report/report_service.rs [hunk 5]
```diff
@@ -192,7 +192,6 @@ pub trait ReportServiceTrait: Sync + Send {
       }}
 
       header {{
```
Comment (mark-prins) [inline server/service/src/report/report_service.rs:194]: another offroad. looks better like this

**Review by clemens-msupply — APPROVED** [review]
Thanks looks good!

**Pull request merged**